### ISSUE TITLE
after system timestamp changes, need to make sure that we write an entry for all tables.

### ISSUE THREAD

**Issue by jopietsch** [netenglabs/suzieq#42]
We want to only serve data that is valid, so we don't want to show data that has polled before the last polled system time.
```
(suzieq) jpiet@t14:/tmp/pycharm_project_1000/suzieq$ python3 /tmp/pycharm_project_1000/suzieq/suzieq/cli/suzieq-cli tables show
Logging to /tmp/suzieq-4oyoeyqw
         table              first_time             latest_time  intervals  latest rows  all rows  datacenters  devices
0        arpnd 2020-03-09 16:54:20.416 2020-03-09 17:20:33.280          9           74       286            1       14
1          bgp 2020-03-09 16:54:20.416 2020-03-09 17:20:33.280          6           32       318            1        9
2           fs 2020-03-09 16:54:20.416 2020-03-09 17:20:33.280          9          229      1504            1       14
3   ifCounters 2020-03-09 16:54:20.416 2020-03-09 17:20:33.280          9          138      5758            1       14
4   interfaces 2020-03-09 16:54:20.416 2020-03-09 17:18:22.208          7          138       496            1       14
5         lldp 2020-03-09 16:54:20.416 2020-03-09 17:18:22.208          8           44       142            1       10
6         macs 2020-03-09 16:54:20.416 2020-03-09 17:20:33.280          8           39       141            1        5
7         mlag 2020-03-09 16:54:20.416 2020-03-09 17:07:26.848          3            4        10            1        4
8       routes 2020-03-09 16:54:20.416 2020-03-09 17:20:33.280          8           24      1376            1       14
9       system 2020-03-09 16:54:20.416 2020-03-09 17:07:26.848          2           14        28            1       14
10        time 2020-03-09 16:54:20.416 2020-03-09 17:20:33.280          9           14        69            1       14
11      topcpu 2020-03-09 16:54:20.416 2020-03-09 17:20:33.280          9           14      2034            1       14
12      topmem 2020-03-09 16:54:20.416 2020-03-09 17:20:33.280          9            9      1477            1        9
13        vlan 2020-03-09 16:54:20.416 2020-03-09 17:07:26.848          3           16        56            1        4
14       TOTAL 2020-03-09 16:54:20.416 2020-03-09 17:20:33.280          9          789     13695            1       14
```
look at topmem, it says that there are 9 latest_rows.
but command topmem show says 6:
```
suzieq) jpiet@t14:/tmp/pycharm_project_1000/suzieq$ python3 /tmp/pycharm_project_1000/suzieq/suzieq/cli/suzieq-cli topmem show
Logging to /tmp/suzieq-luzfqijc
  datacenter  hostname               timestamp
1   dual-bgp    exit02 2020-03-09 17:14:00.064
2   dual-bgp  internet 2020-03-09 17:16:11.136
4   dual-bgp    leaf02 2020-03-09 17:14:00.064
5   dual-bgp    leaf03 2020-03-09 17:09:37.920
6   dual-bgp    leaf04 2020-03-09 17:11:48.992
7   dual-bgp   spine01 2020-03-09 17:09:37.920
```
the problem is that in table show, I'm directly calling get_table_df, not get_valid_df, and so it's not doing the merge with the system table.
here is the system table
```
uzieq) jpiet@t14:/tmp/pycharm_project_1000/suzieq$ python3 /tmp/pycharm_project_1000/suzieq/suzieq/cli/suzieq-cli system show
Logging to /tmp/suzieq-uyzotg20
   datacenter   hostname model      version   vendor architecture status          address          uptime               timestamp
1    dual-bgp     edge01    vm  16.04.6 LTS   Ubuntu       x86-64  alive  192.168.121.165 00:17:30.848000 2020-03-09 17:07:26.848
3    dual-bgp     exit01    vm       3.7.12  Cumulus       x86-64  alive  192.168.121.108 00:17:25.848000 2020-03-09 17:07:26.848
5    dual-bgp     exit02    vm       3.7.12  Cumulus       x86-64  alive  192.168.121.236 00:17:28.848000 2020-03-09 17:07:26.848
7    dual-bgp   internet    vm       3.7.12  Cumulus       x86-64  alive  192.168.121.237 00:17:30.848000 2020-03-09 17:07:26.848
9    dual-bgp     leaf01    vm       3.7.12  Cumulus       x86-64  alive   192.168.121.86 00:17:28.848000 2020-03-09 17:07:26.848
11   dual-bgp     leaf02    vm       3.7.12  Cumulus       x86-64  alive   192.168.121.22 00:17:25.848000 2020-03-09 17:07:26.848
13   dual-bgp     leaf03    vm       3.7.12  Cumulus       x86-64  alive  192.168.121.132 00:17:29.848000 2020-03-09 17:07:26.848
15   dual-bgp     leaf04    vm       3.7.12  Cumulus       x86-64  alive   192.168.121.53 00:17:31.848000 2020-03-09 17:07:26.848
17   dual-bgp  server101    vm  16.04.6 LTS   Ubuntu       x86-64  alive  192.168.121.235 00:22:13.848000 2020-03-09 17:07:26.848
19   dual-bgp  server102    vm  16.04.6 LTS   Ubuntu       x86-64  alive  192.168.121.239 00:22:54.848000 2020-03-09 17:07:26.848
21   dual-bgp  server103    vm  16.04.6 LTS   Ubuntu       x86-64  alive   192.168.121.83 00:22:23.848000 2020-03-09 17:07:26.848
23   dual-bgp  server104    vm  16.04.6 LTS   Ubuntu       x86-64  alive   192.168.121.54 00:22:14.848000 2020-03-09 17:07:26.848
25   dual-bgp    spine01    vm       3.7.12  Cumulus       x86-64  alive  192.168.121.242 00:17:30.848000 2020-03-09 17:07:26.848
27   dual-bgp    spine02    vm       3.7.12  Cumulus       x86-64  alive  192.168.121.105 00:17:29.848000 2020-03-09 17:07:26.848
```
so there are three entries in latest topmem that don't get shown because their time is older than the last time system table was queried, which is this line in get_valid_df
```.query("timestamp_x >= timestamp_y")```
if I comment out that one line, then I get
```(suzieq) jpiet@t14:/tmp/pycharm_project_1000/suzieq$ python3 /tmp/pycharm_project_1000/suzieq/suzieq/cli/suzieq-cli topmem show
Logging to /tmp/suzieq-gtrmtk3f
  datacenter  hostname               timestamp
0   dual-bgp    exit01 2020-03-09 16:54:20.416
1   dual-bgp    exit02 2020-03-09 17:14:00.064
2   dual-bgp  internet 2020-03-09 17:16:11.136
3   dual-bgp    leaf01 2020-03-09 16:54:20.416
4   dual-bgp    leaf02 2020-03-09 17:14:00.064
5   dual-bgp    leaf03 2020-03-09 17:09:37.920
6   dual-bgp    leaf04 2020-03-09 17:11:48.992
7   dual-bgp   spine01 2020-03-09 17:09:37.920
8   dual-bgp   spine02 2020-03-09 16:54:20.416
```
in other words, if you look at exit01, it didn't show up in the first topmem show I sent. That is because exit01 ahs a timestamp of 16:54 in topmem but has a timstamp of 17:07 in system.

I looked more, and the reason that the system table has an updated entry again is because I restarted the poller, not that the data had changed. I don't know why topmem wasn't updated after I restarted the poller.
